custom/icds_reports/tests/reports/test_awc_reports.py

```diff
@@ -2540,7 +2542,27 @@ def test_pregnant_details_first_record_second_trimester(self):
             )
             self.assertEqual(
                 data['data'][1],
-                []
```

Comment:
This has changed Because Lukaz added a filter to consider only the data which has home_visit_date(which is same as bp_date) set and It was not being set earlier because fixture was not there.  I added the row in the fixture to pass the test cases. Therefore now home_visit_date is set. Hence, this change

Reply:
not sure what you meant by `It looks liek the same case is loaded twice`